FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift

```diff
@@ -262,6 +279,18 @@ extension ThreadDetailsViewController {
         }
     }
 
+    private func tryToFetchSenderPubKeys(message: Message, folder: String, indexPath: IndexPath) {
+        Task {
+            let processedMessage = try await messageService.getAndProcessMessage(
```

Comment:
What happens to exceptions that are thrown in `Task` and not catched? Do they crash the app or are they ignored? Any logs?

Either way, some exception handling will be necessary. If lookup or something else fails, the signature badge will be stuck in loading state forever. For these situations, we may need a badge like `Failed to verify signature due to: \(error)`.